AlembicImporter/Packages/com.unity.formats.alembic/Documentation/com.unity.formats.alembic.md

```diff
@@ -194,13 +207,13 @@ The effect the recording press "Play". The ABC file will be recorded to the outp
 
 ## Assigning Materials on Import
 
-By default the Alembic will import Meshes using the `Default Material`. So, you will need to manually reassign your Materials for each object.
+By default Unity will assign the `Default Material` to imported Alembic Meshes. You will need to manually reassign your Materials for each object.
 
 The Alembic package does not support remapping Face Set names to Materials or creating Materials from Face Set names.
 
-## Alembic Materials
+## Alembic Shaders
 
-The Alembic package includes the following Materials:
+The Alembic package includes the following Shaders:
 
```

Comment:
Let's make sure we have the complete inventory here, in case more have been added since the docs were last updated. Can you or @alexismorin dig in and find them all?

Reply:
these are all of user visible shaders (by user visible I mean available in the Material shader menu):
![image](https://user-images.githubusercontent.com/2340643/42042194-9fc11306-7ac1-11e8-9885-29f5c4a0c18b.png)

Reply:
perfect!